### DISCUSSION TITLE
Function location will not change

### DISCUSSION THREAD

**Discussion by DevMaxC** [vercel/community#1994] [Help] [answered]
### Summary

Im attempting to change my function location in my project's settings, however in the build logs, its showing as somewhere different to where ive chose, ive rebuilt many times and waited over a day. Still not changed :/

**Comment by amyegan** [comment]
Hi @DevMaxC! I want to make sure I understand you correctly and don't suggest something you already tried. Can you share some screenshots or a [minimal reproduction](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) and walk me through the steps you took?

**Reply by DevMaxC** [reply]
![image](https://user-images.githubusercontent.com/20259746/230697222-2b81027b-78d7-46f6-8d7d-5b6c6a42d510.png)
![image](https://user-images.githubusercontent.com/20259746/230697244-415375b0-2d40-45e9-b231-2fbaeba1c419.png)

I must be doing something stupid. But I don't know. Is there any config files where server location can be defined that I may be missing. There's some turbo repo boilerplatey in my code so maybe there is a hidden config file I haven't spotted.

**Reply by amyegan** [reply, marked as answer]
Thanks for clarifying, Max. It looks like you're doing everything right! We don't run builds in every region we provide, so the closest available build location is used. Rest assured that this build location is **not** the same as the region where your Serverless Functions will execute.
